Feature/teams

talorlangnas-moveo opened the pull request:
done

talorlangnas-moveo committed:
create TeamCard component

talorlangnas-moveo committed:
last commit before changing

talorlangnas-moveo committed:
update due to meetings instructions

- remove custom hooks and unused scss files.
- create LeagueTab component
- create LeaguesResult component
- create LeaguesLobby component
- adding service files

talorlangnas-moveo committed:
final commit

talorlangnas-moveo committed:
Update LeagueTab.scss

removed box shadow

DanielBaronMoveo commented on src/assets/styles/components/LeagueTab.scss:
```diff
@@ -0,0 +1,59 @@
+  padding: 0.75rem;
+  background: #fff;
+  border-radius: 0.5rem;
+  // box-shadow: 0 1px 4px rgba(0, 0, 0, 0.1);
```
Remove commented code.

talorlangnas-moveo replied:
Done

DanielBaronMoveo commented on src/assets/styles/components/LeaguesLobby.scss:
```diff
@@ -0,0 +1,14 @@
+.flex-container {
+  display: flex;
+  flex-direction: column;     // ✅ stack children vertically
+  // align-items: center;        // ✅ center horizontally
+  justify-content: flex-start; // ✅ (optional) top align
+  min-height: 100vh;          // optional: make it fill the screen
```
Remove commented code, also those that were meant to describe the styling.

talorlangnas-moveo replied:
Done

DanielBaronMoveo commented on src/components/TeamsResult.tsx:
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import TeamCard from './TeamCard';
+import type { Team } from '../types/interfaces/Team';
+// import '../assets/styles/components/TeamsResults.scss';
```
Remove commented code.

talorlangnas-moveo replied:
Done

DanielBaronMoveo commented on src/main.tsx:
```diff
@@ -1,6 +1,6 @@
 import { StrictMode } from 'react'
 import { createRoot } from 'react-dom/client'
-import './index.css'
+// import './index.css'
```
Remove commented code.

talorlangnas-moveo replied:
Done

DanielBaronMoveo commented on src/services/sportsApi.ts:
Are we using this file? I'm asking because I don't see any component using it. 
If it's irrelevant let's remove it.

talorlangnas-moveo replied:
Done

DanielBaronMoveo commented on src/services/league.service.ts:
Do we really need all the following functions within the service? I can't see if we are using them.

talorlangnas-moveo replied:
Done

DanielBaronMoveo reviewed (changes requested):
Good job, it looks really good. Thumbs up for the effort. 🚀

Few things that I want you to go over:
- Minor changes, commented code cleanups.
- Mainly, we don't want to create over logical stuff and not use them so if you find some functions that we are not using irrelevant let's remove them. You can keep it if you want to but for the best practices I recommend in the future we only include what's necessary for us.
- **For every comment mentioned in the PR we are going to reply back to the reviewer with a 'Done' or some indicator so the reviewer (myself) will know you took care of his comment.** 

If something is unclear let me know! :)

talorlangnas-moveo committed:
fix the comments issue

- Remove commented code from: src/assets/styles/components/LeagueTab.scss

- Remove commented code from:
src/assets/styles/components/LeaguesLobby.scss

- Remove commented code from:
src/main.tsx

- Remove the file src/services/sportsApi.ts

- Remove unused functions from:
src/services/league.service.ts
